phpunit.xml.dist

```diff
@@ -1,30 +1,18 @@
 <?xml version="1.0" encoding="UTF-8"?>
-
```

Comment:
It looks like only the indentation changed. You can keep it as it was to minimize the diff size and ease review.